client/my-sites/plans/current-plan/current-plan-thank-you/backup-thank-you.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React, { Fragment } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ThankYou from './thank-you';
+
+const BackupProductThankYou = ( { translate } ) => (
+	<ThankYou
+		illustration="/calypso/images/illustrations/security.svg"
+		showCalypsoIntro
+		showContinueButton
+		title={ translate( 'Hello backups!' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'So long spam, hello backups!' )` *ES Score: 7*